lms/djangoapps/sneakpeek_deeplink/tests.py

```diff
@@ -73,6 +80,11 @@ def setUp(self):
             pref_key="allow_nonregistered_access",
             pref_value="true",
         ).save()
+        CoursePreference(
+            course_id=self.course_to_delete.id,
+            pref_key="allow_nonregistered_access",
```

Comment:
As above, should I apply this change to the course preference creations that existed before my changes?